lib/log.rb

```diff
@@ -80,6 +80,19 @@ def initialize(level=nil)
       else
         @level = INFO
       end
+
+      # Reference variables to evaluate custom error values.
+      @custom_err = ''
```

Comment:
```suggestion
      handled_error = ''
```

This will use the reader method defined by `attr_accessor`.

Reply:
But in fact it would be a good ruby style to remove this line and write a method like

```ruby
def handled_error?
  handled_error && !handled_error.empty?
end
```

Then later, instead of checking `logger.custom_errmsg.length > 0`, you would do:
```ruby
logger.handled_error?
```

Reply:
Updated